actors/common/src/main/java/org/sunbird/learner/util/Util.java

```diff
@@ -600,6 +602,11 @@ public static SearchDTO createSearchDto(Map<String, Object> searchQueryMap) {
         search.setLimit(((BigInteger) searchQueryMap.get(JsonKey.LIMIT)).intValue());
       }
     }
+    if (search.getLimit() > MAXIMUM_ELASTIC_LIMIT
```

Comment:
We need to handle both separately.
1. if limit is more than max then set it max and offset as zero 
2. if limit+offset is more than max then reduce offset to reach max only.